.storybook/preview.tsx

```diff
@@ -22,44 +23,34 @@ const queryClient = new QueryClient({
 
 queryClient.setQueryData(['version'], '1.2.3')
 
-const preview: Preview = {
-  parameters: {
-    actions: { argTypesRegex: '^on[A-Z].*' },
-    controls: {
-      matchers: {
-        color: /(background|color)$/i,
-        date: /Date$/i,
-      },
+export const parameters = {
+  eyes: {
+    beforeCaptureScreenshot: async () => {
```

Comment:
waitBeforeCapture?

Reply:
the before capture screenshot hook has a different purpose - it runs the script inside the rendering environment (UFG), so waiting for resources there doesn't make sense as all resources have already been captured in the client side.
I'll try to make it work